cartographer_ros/cartographer_ros/node.cc

```diff
@@ -421,7 +422,7 @@ void Cartographer::LaunchSubscribers(const TrajectoryOptions& options,
     subscribers_[trajectory_id].push_back(
         {SubscribeWithHandler<sensor_msgs::msg::MultiEchoLaserScan>(
              &Cartographer::HandleMultiEchoLaserScanMessage, trajectory_id, topic,
-             node_handle_, this, rmw_qos_profile_default),
+             node_handle_, this),
```

Comment:
Not in the scope of this PR, but I find it interesting that the default QoS was used here, the sensor profile may have made more sense.

Reply:
See #29, which changes this to the sensor profile :).